packages/cli/src/commands/doctor/healthchecks/index.ts

```diff
@@ -1,5 +1,6 @@
 import nodeJS from './nodeJS';
 import {yarn, npm} from './packageManagers';
+import java from './jdk';
```

Comment:
let's stick with `jdk`, that's what we install :) 
```suggestion
import jdk from './jdk';
```